### PR TITLE
add repeat to list of Modules 

### PR DESCRIPTION
**Pull request by musab**
repeat is a module to repeat a given string n times

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ __NOTICE__: Deno has a few official modules that could be found at [deno_std](ht
 - [textproto](https://github.com/denoland/deno_std/tree/master/textproto)
 - [watch](https://github.com/jinjor/deno-watch) - A file watcher.
 - [wu-diff-js](https://github.com/bokuweb/wu-diff-js) - A diff library to compute differences between two slices using wu(the O(NP)) algorithm.
+- [repeat](https://github.com/musab/repeat) - Repeats the given string n times.
```
Comment (zhmushan) [inline README.md]: Should be sorted alphabetically

**Comment by kevinkassimo** [comment]
Hmm my slight concern is that we already have [`String.prototype.repeat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat) in the language standard...

**Comment by musab** [comment]
> Hmm my slight concern is that we already have [`String.prototype.repeat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat) in the language standard... and it is supported in all major browsers except for IE already...

Ah, that's news to me... I see your point...

**Comment by kevinkassimo** [comment]
> Ah, that's news to me... I see your point...
In that case, I would close this PR as I want to avoid causing confusion to others. Feel free to reply if you think there is a point we should prefer this over the standard one.

**Pull request closed without merging**